sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -39,7 +39,21 @@ import org.apache.spark.util.Utils
 
 
 /**
- * Functions available for DataFrame operations.
+ * Commonly used functions available for DataFrame operations. Using functions defined here provides
+ * a little bit more compile-time safety to make sure the function exists.
+ *
+ * Spark also includes more built-in functions that are less common and are not defined here.
+ * You can still access them (and all the functions defined here) using the [[functions.expr()]] API
```

Comment:
```
[error] /home/jenkins/workspace/SparkPullRequestBuilder/sql/core/target/java/org/apache/spark/sql/functions.java:7: error: unexpected text
[error]  * You can still access them (and all the functions defined here) using the {@link functions.expr()} API
[error]                                                                             ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder/sql/core/target/java/org/apache/spark/sql/functions.java:9: error: unexpected text
[error]  * the latest version of Spark at {@link https://spark.apache.org/docs/latest/api/sql/index.html}.
[error]                                   ^
```

Seems both links are the problem in Javadoc. Shall we just use `` `functions.expr() `  `` and leave the `https://spark.apache.org/docs/latest/api/sql/index.html` like without `[[...]]`?